index.html

```diff
@@ -2,10 +2,11 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8" />
-        <link rel="icon" href="/favicon.ico" />
+        <link rel="icon" type="image/svg+xml" href="/favicon/favicon.svg">
+        <link rel="icon" type="image/png" href="/favicon/favicon.png">
```

Comment:
I'll look into declaring a manifest or something like that, it could help the browser decide which one to load and not do this double/triple load at startup